.github/workflows/versioning.yaml

```diff
@@ -0,0 +1,135 @@
+name: versioning
```

Comment:
A non-issue, but this file can be deleted.

https://greene-lab.gitbook.io/lab-website-template-docs/basics/repo-structure#template-repo-only-content

Reply:
Thanks - I'll remove this!